packages/components/tests/dummy/app/routes/foundations/colors.js

```diff
@@ -1,9 +1,9 @@
 import Route from '@ember/routing/route';
 
-import TOKENS_RAW from '@hashicorp/design-system-tokens/dist/docs/products/tokens.json';
+// import TOKENS_RAW from '@hashicorp/design-system-tokens/docs/products/tokens.json';
```

Comment:
@Dhaulagiri continuation of other comment ^^ here's the other thing: 

(2) I do get an ember-auto-import webpack error here. Since I was having a bunch of strange yaml issues (still confused why) did this to decouple things

I think this is higher priority and the Percy thing could be a fast follow, let me know if you agree. I will work on this first when back at key

Reply:
it looks like you figured this out though?

Reply:
disregard, i see the action is still failing...

Reply:
Sorry for the confusion, pushed one last commit to ensure it was as I expected (following all of the other changes in this PR)

I think it could arguably be merged with a temporary stopgap without tokens and colors in the website as an incremental step forward, but can see if get anywhere before tmrw morning. Feel free to drop the last commit https://github.com/hashicorp/design-system/pull/60/commits/c5300cb8e88e0b4038cfed3a9f0c42f5f8c5d1e6 and merge in the morning if there is no progress

I'm confused why at `design-system/packages/components` a `yarn test:ember` passes, but the ember try's don't